resources/builder.inc.sh

```diff
@@ -359,7 +359,7 @@ _builder_failure_trap() {
 # finishes.
 #
 _builder_cleanup_deps() {
-  if ! builder_is_dep_build && [[ ! -z ${_builder_deps_built+x} ]]; then
+  if ! builder_is_dep_build && ! builder_is_child_build && [[ ! -z ${_builder_deps_built+x} ]]; then
```

Comment:
@jahorton this fixes the issue you reported to me on Slack. Dependency report was being deleted as child builds finished.